adobe_vipm/flows/fulfillment/shared.py

```diff
@@ -1219,17 +1221,22 @@ def __call__(self, client, context, next_step):
             context.authorization_id,
             context.adobe_customer_id,
         )
+
+        adobe_subscriptions_in_agreement = get_adobe_subs_from_agreement_subscriptions(
```

Comment:
Could you pls revert back this changes? I mean, I need to think how to better refactor it